operators/tokenizer/bpe_kernels.cc

```diff
@@ -148,6 +148,8 @@ std::vector<int64_t> KernelBpeTokenizer::Tokenize(ustring& input,
     if (IsUnicodeSpace(str.back())) {
       str.pop_back();
     }
+    // remove newlines as CLIP ignores them
+    str.erase(std::remove(str.begin(), str.end(), '\n'), str.end());
```

Comment:
they are removed or are treated as a WS?

Reply:
same thing in CLIP as there is WS clean in CLIPTokenizer: https://github.com/huggingface/transformers/blob/v4.34.0/src/transformers/models/clip/tokenization_clip.py#L94